app/routers/event.py

```diff
@@ -84,6 +90,60 @@ async def eventedit(request: Request) -> Response:
     return templates.TemplateResponse("eventedit.html", {"request": request})
 
 
+@router.get("/edit/view_countries", include_in_schema=False)
+async def choose_country(request: Request,
+                         session=Depends(get_db)) -> Response:
+    """
+    Displays the list of all countries name from "Country" table.
+    """
+    countries_names = get_all_countries_names(session=session)
+    return templates.TemplateResponse("eventedit.html",
+                                      {"request": request,
+                                       "countries_names": countries_names})
+
+
+@router.post("/edit/view_countries", include_in_schema=False)
+async def check_country_time(request: Request,
+                             session=Depends(get_db)) -> Response:
+    """
+    Displays datalist of all countries name from "Country" table.
+    By clicking "Check time" the converted meeting time is displayed.
+    If the user's ip is not recognized, an error message will appear.
+    ** temporarily using a random israeli ip address instead **
+    """
+    # TODO: define ip as request.client.host
+    # ip = request.client.host
```

Comment:
Why is this one commented out?

Reply:
In order to let the tests pass, If I understood correctly the current IP is not related to any geographical location